src/wp-includes/default-constants.php

```diff
@@ -325,9 +325,10 @@ function wp_cookie_constants() {
 
 	/**
 	 * @since 2.0.0
+  	 * @since 6.4.0 The default value is empty.
```

Comment:
Couple of nitpicks about the documentation:
1. If this is being documented, , the docblock should be above the `define()`, not above the `if ()`. You are, after all, documenting the constant, not the condition.
2. If this is being documented, please actually document _what has changed_. "The default value is empty." is ambiguous. What is empty ? What do you mean by default value ? (constants cannot be changed once defined)

Suggestion for the text: "The standard value has changed from `false` to an empty string".

You may also want to add a `@var string` tag to indicate that the constant is expected to contain a string value (and no other type).